roles/cis/tasks/6/verify/6.1.3.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: "6.1.3 Verify permissions on /etc/shadow are configured (Scored)"
+  block:
+    - name: "6.1.3 verify permissions"
+      stat:
+        path: /etc/shadow
+      register: etcshadow
+    - fail:
+        msg: "6.1.3 failed to verify uid. expected: 0. found: {{ etcshadow.stat.uid }}"
+      when: etcshadow.stat.uid != 0
+    - fail:
+        msg: "6.1.3 failed to verify gid. expected: 0. found: {{ etcshadow.stat.gid }}"
+      when: etcshadow.stat.gid != 0
+    - fail:
+        msg: "6.1.3 failed to verify permissions. expected: 0000. found: {{ etcshadow.stat.mode }}"
+      when: etcshadow.stat.mode != "0000"
+  tags:
+    - scored
```

Comment:
please add the cis and cis.6 tags here as well. see PR: https://github.com/vmware/ansible-security-hardening/pull/4

Reply:
fixed in https://github.com/vmware/ansible-security-hardening/pull/5/commits/c176df4e250ba9e907e1caf7d24433fd1ae6055c